presentation-api/cast-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
@RishabhGurjar09 Why do we need two apps? Can't be controlled by a separate page/route within the same app?